Fix incorrect description of call pseudoinstruction expansion (#214)
riscv-non-isa/riscv-elf-psabi-doc: riscv-elf.adoc

behavior, but that has turned out to be unnecessary, and they are now
interchangeable.

With relaxation enabled, the `AUIPC` instruction in the `AUIPC+JALR` pair has

> both a `R_RISCV_CALL` or `R_RISCV_CALL_PLT` relocation and a `R_RISCV_RELAX`

Reviewer: ```suggestion
both a `R_RISCV_CALL` or `R_RISCV_CALL_PLT` relocation and an `R_RISCV_RELAX`
```

[,asm]
----
    call symbol

>     call symbol@PLT

Reviewer: ```suggestion
    call symbol@plt
```

    jalr  ra, ra, 0
----

> and when symbol with `@PLT` suffix it expands to:

Reviewer: ```suggestion
and when symbol has an `@plt` suffix it expands to:
```
(yes it is actually case-sensitive in both assemblers)
